app/scripts/constants.js

```diff
@@ -64,6 +64,33 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
   CLI: {
     "Latest Release":          "https://access.redhat.com/downloads/content/290"
   },
+
+  // Change the minishift link to the CDK for enterprise.
+  CATALOG_HELP_RESOURCES: {
+    links: [
+      {
+        title: 'Documentation',
+        help: ''
+      },
+      {
+        title: 'Interactive Learning Portal',
+        href: 'https://learn.openshift.com'
+      },
+      {
+        title: 'Red Hat Container Development Kit',
```

Comment:
feels a bit wordy compared to the other links, can we drop the "Red Hat" from it?  @sspeiche ?

Reply:
> feels a bit wordy compared to the other links, can we drop the "Red Hat" from it?

nudge @sspeiche :)